### PR TITLE
Add supports html: false to new website blocks.

### PR DESCRIPTION
**Pull request by Soean**
## Description
Disables the HTML mode for the new website editor blocks.

### CODE REVIEW

**Review by epiqueras — CHANGES_REQUESTED** [review]
Can we add these in the `block.json` files instead?

**Comment by Soean** [comment]
In Gutenberg we only use `name`, `category` and `attributes` in block.json at the moment.

See also:
>We plan to deprecate this field. In the long run, we want to provide a more robust API to achieve the same goal. It would be nice to skip this field from the REST API. We keep this field on the client for all core blocks.
@gziolo at https://core.trac.wordpress.org/ticket/47620#comment:11

File: packages/block-library/src/template-part/index.js [hunk 1]
```diff
@@ -14,5 +14,8 @@ export { metadata, name };
 
 export const settings = {
 	title: __( 'Template Part' ),
+	supports: {
+		html: false,
```
Comment (epiqueras) [inline packages/block-library/src/template-part/index.js]: This one already has it in `block.json`, we should remove it there.

**Review by epiqueras — CHANGES_REQUESTED** [review]

File: packages/block-library/src/template-part/index.js (hunk 1 above)
Comment (Soean) [reply packages/block-library/src/template-part/index.js]: Oh right. I removed it.

**Review by epiqueras — APPROVED** [review]

**Pull request merged**